src/bindings/python/src/openvino/runtime/ie_api.py

```diff
@@ -242,6 +242,28 @@ def create_infer_request(self) -> InferRequest:
         """
         return InferRequest(super().create_infer_request())
 
+    def query_state(self) -> None:
+        """Gets state control interface for the underlaying infer request.
```

Comment:
```suggestion
        """Gets state control interface for the compiled model.
```

Reply:
I am not sure if that's proper change. The whole goal of CompiledModel simplified interface is to hold InferRequest in it. Saying that CompiledModel will get/reset state "on itself" might be misleading to the users/devs.

I am personally leaning into leaving the message as-is. Even if on the outside it looks like CompiledModel is doing all work -- there are still no clear messages for "Model-centric approach" in main OV API. If such approach shift happens, then I would be first to reword documentation to the exact wording proposed by you.

@akuporos could you share your thoughts on that?

Reply:
@jiwaszki I agree with you.
I also think that it may sound confusing.